src/main/java/gregtech/common/mui/widget/workbench/CraftingInputSlot.java

```diff
@@ -76,15 +76,13 @@ public void draw(GuiContext context, WidgetTheme widgetTheme) {
         ItemStack itemstack = this.syncHandler.getStack();
         if (itemstack.isEmpty()) return;
 
-        if (!this.hasIngredients) {
-            RenderUtil.renderRect(0, 0, 18, 18, 100, 0x80FF0000);
-        }
-
-        guiScreen.setZ(100f);
-        guiScreen.getItemRenderer().zLevel = 100f;
-        RenderUtil.renderItemInGUI(itemstack, 1, 1);
         guiScreen.getItemRenderer().zLevel = 0.0F;
         guiScreen.setZ(0f);
+        RenderUtil.renderItemInGUI(itemstack, 1, 1);
```

Comment:
these z index calls are no longer necessary, and in fact the reference to the GuiScreenWrapper can be removed as well. it's not blocking though, it can be fixed later